### PR TITLE
Fill the input field when clicked on balance

### PR DESCRIPTION
**Pull request by Sharqiewicz**
🦈 🟢 ready
---
- [x] fix hover on select token
- [x] implement click on balance to set max amount

### CODE REVIEW

File: src/pages/swap/sections/BankDetails.tsx [hunk 1]
```diff
@@ -16,11 +16,16 @@ export const BankDetails: FC<BankDetailsProps> = ({ registerBankAccount, registe
     <p className="my-5 font-thin text-center text-gray-400">Which bank account should we send the funds to?</p>
     <LabeledInput
       label="PIX ID"
```
Comment (TorstenStueber) [inline src/pages/swap/sections/BankDetails.tsx:18]: This whole component is not even used anymore. We should be able to remove the whole file.

**Review by TorstenStueber — APPROVED** [review]
👍 

Code is all good, just that one component can be removed.

File: src/pages/swap/sections/BankDetails.tsx (hunk 1 above)
Comment (gianfra-t) [reply src/pages/swap/sections/BankDetails.tsx:18]: What is this component about?

**Review by prayagd — APPROVED** [review]
LGTM works as expected

**Review by ebma — APPROVED** [review]

**Pull request merged**